### PR TITLE
chore(ojoi): Renaming files/code to correct names

### PR DESCRIPTION
**Pull request by jonbjarnio**
## What

- Renaming files and modules to correct names.

## Why

To keep consistency within the codebase

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

## Summary by CodeRabbit

- **New Features**
  - Updated API endpoint paths for better consistency and accuracy.
  - Added new definitions for `AdvertAttachment` in the client configuration to enhance data handling.

- **Refactor**
  - Renamed various entities and modules from "Ministry of Justice" to "Official Journal of Iceland" to reflect the updated organizational structure.
  - Changed import paths and references to align with the new naming conventions.

- **Documentation**
  - Adjusted project and display names in configuration files for clarity and consistency across platforms.

### CODE REVIEW

File: libs/api/domains/official-journal-of-iceland/.eslintrc.json
Comment (saevarma) [inline libs/api/domains/official-journal-of-iceland/.eslintrc.json:1]: You should add your team as the codeowner for this library.

File: libs/clients/official-journal-of-iceland/public/.eslintrc.json
Comment (saevarma) [inline libs/clients/official-journal-of-iceland/public/.eslintrc.json:1]: You should add your team as codeowner of this library.

**Review by saevarma — APPROVED** [review]
Core files LGTM

**Review by Valur — APPROVED** [review]

**Review by lodmfjord — APPROVED** [review]
LGTM

**Review by Toti91 — APPROVED** [review]

**Comment by jonbjarnio** [comment]
@coderabbitai resolve

**Pull request merged**